data/json/hobbies.json

```diff
@@ -625,7 +1059,14 @@
     "description": "Your time in the rink has taught you that sometimes the best way forward is straight through whoever's trying to block you.",
     "points": 3,
     "skills": [ { "level": 2, "name": "bashing" }, { "level": 2, "name": "swimming" }, { "level": 1, "name": "melee" } ],
-    "traits": [ "PROF_SKATER" ]
+    "traits": [ "PROF_SKATER" ],
+    "difficulty": [
+      [ "combat", "easy" ],
+      [ "mobility", "normal" ],
```

Comment:
```suggestion
      [ "mobility", "easy" ],
```

Reply:
Just the skater trait is enough (even if barely) for a good mobility rating, its helps a lot if you manage to find a skate.